.github/workflows/release.yaml

```diff
@@ -0,0 +1,79 @@
+name: create release on new tags
+
+# This workflow is triggered by new version tags (e.g. "1.12.2") on the main branch.
+#
+# Tagging should be automatic after each merged PR release (see the build.yaml workflow). It can
+# also be manually triggered if needed. Note that the tests are not run by this workflow, so
+# release manually at your own risk. If manually tagging the main branch, the tag must match the
+# version set in ./pkg/chartverifier/version/version_info.json.
+#
+# This workflow builds the tarball, creates the Github release and attaches the tarball to it.
+
+on:
+  push:
+    # Publish semver tags as releases.
+    tags: '[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  build-and-release:
+    name: Create GitHub release
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: go.mod
+
+      - name: Print tag to GITHUB_OUTPUT
+        id: get_tag
+        run: |
+          echo "release_version=${GITHUB_REF#refs/*/}" | tee -a $GITHUB_OUTPUT
```

Comment:
Maybe it's being too cautious, but should we also consider printing out the `git rev-parse HEAD` output?

Reply:
I see you already resolved this conversation ? (intended ?)

I don't think it's needed to print the commit ID, unless you have a strong opinion otherwise